frontend/lib/norent/letter-builder/letter-preview.tsx

```diff
@@ -56,11 +60,34 @@ const SendLetterModal: React.FC<{
   );
 };
 
+const ForeignLanguageOnly: React.FC<{ children: React.ReactNode }> = (
+  props
+) => {
+  const isForeignLanguage = i18n.locale !== "en";
+
+  if (!isForeignLanguage) return null;
+
+  return <>{props.children}</>;
+};
+
+const Microcopy: React.FC<{ children: React.ReactNode }> = (props) => (
+  <p className="is-uppercase is-size-7">{props.children}</p>
+);
```

Comment:
It's possible we're really using React instead of CSS here, but it sorta makes the code more readable?

Reply:
Hmmm yeah this is a tough one. I personally don't like to make react components that are just purely styling related, because when I see the custom component I would assume that it's doing something more complicated than just styling the element, and would then need to take the extra step of tracing back to the definition of the component to verify. But, of course this does make things more DRY and everyone has their own preferences. Totally your call on this!